src/wtr-config.js

```diff
@@ -0,0 +1,106 @@
+import { defaultReporter } from '@web/test-runner';
+import { playwrightLauncher } from '@web/test-runner-playwright';
+import { visualDiff } from './visual-diff-plugin.js';
+import { visualDiffReporter } from './visual-diff-reporter.js';
+
+const DEFAULT_PATTERN = type => `./**/*.${type}.js`;
+const DEFAULT_VDIFF = true;
```

Comment:
I think we'll want this to be `false` by default since visual diff will still require some configuration within a new repo.